components/autofill/src/sync/mod.rs

```diff
@@ -250,4 +250,209 @@ pub mod test {
 }
 
 #[cfg(test)]
-mod tests;
+mod tests {
```

Comment:
A while back I moved these tests into `test_mod.rs`, and this adds them back here - which I'd be fine with if you felt it better, except for the fact that it didn't touch `test_mod.rs`, so it looks like this has just duplicated them.